client/signup/steps/theme-selection/index.jsx

```diff
@@ -58,14 +92,38 @@ module.exports = React.createClass( {
 		);
 	},
 
+	renderThemePreview() {
+		if ( this.state.previewTheme ) {
+			return (
+				<ThemePreview
+					showPreview={ true }
```

Comment:
I originally tried to do that, but it was causing errors because `theme={  }` would be `null`. I guess I can pick one of the themes by default so it renders properly, and have a separate flag for visibility.